commands/summary.go

```diff
@@ -274,6 +235,103 @@ func (cmd *summaryCommand) makeArchive() (err error) {
 	return //nolint:nakedret
 }
 
+type profilerPath struct {
+	suffix  string
+	webPath string
+}
+
+func (cmd *summaryCommand) getPprofData() []string {
+	profilerPaths := []profilerPath{
+		{
+			suffix:  "profile.pb.gz",
+			webPath: "/profile?seconds=60",
+		},
+		{
+			suffix:  "heap.pb.gz",
+			webPath: "/heap?gc=1",
+		},
+		{
+			suffix:  "trace.out",
+			webPath: "/trace?seconds=10",
+		},
+	}
+	apps := map[string]string{
+		"pmm-agent": "http://127.0.0.1:7777/debug/pprof",
+	}
+	if !cmd.SkipServer {
+		apps["pmm-managed"] = "http://127.0.0.1:7773/debug/pprof"
+		apps["qan-api2"] = "http://127.0.0.1:9933/debug/pprof"
+	}
+
+	out := make(chan string)
+	files := make([]string, 0)
+	wg := &sync.WaitGroup{}
+	wgr := &sync.WaitGroup{}
```

Comment:
That's the worst offender of the zero value rule: there are no fields one can initialise in this struct; it is not even important that it is a struct.

Reply:
That wasn't addressed.